Sources/Executables/TokenCodegenGenerator/codegen.swift

```diff
@@ -203,13 +215,26 @@ private func generateSemanticVariable(
 
     return DeclSyntax(
         """
+        #if os(macOS)
+        public static let \(raw: variableName) = NSColor(
+            name: nil,
+            dynamicProvider: {
+                if $0.name == .aqua || $0.name == .vibrantLight || $0.name == .accessibilityHighContrastAqua || $0.name == .accessibilityHighContrastVibrantLight {
```

Comment:
For now I am only differentiate between light and dark, in a future we can consider differentiate between more appearances. To be discussed with the design team probably.